src/styles/main.scss

```diff
@@ -1,3 +1,117 @@
-body {
+@import "variables";
+@import "stars";
+@import "header";
+@import "nav";
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+.card {
+  background: $background--color--card;
+  border: $border--thickness solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px;
+  width: 200px;
+  transition: transform $transition-duration;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 32px auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $color--text;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    display: flex;
+    font-size: 10px;
+    color: $color--text--title;
+    font-weight: 400;
+    margin-bottom: 16px;
+    align-self: center;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__reviews {
+    display: flex;
+    font-size: 10px;
+    color: $color--text;
+    font-weight: 400;
+    margin-bottom: 26px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: $color--text;
+    }
+
+    &-title {
+      font-size: 12px;
+      line-height: 18px;
+      color: $color--text--title;
+      font-weight: 400;
+    }
+  }
+
+  &__button {
+    background: $color--button;
+    border: $border--thickness solid $color--button;
+    border-radius: 5px;
+    height: $height--button;
+    width: $width--button;
+    text-transform: uppercase;
+    font-weight: 700;
+    line-height: 16px;
+    color: #fff;
+    margin-bottom: 16px;
+    font-size: 14px;
+
+    &:hover {
+      transition: background-color $transition-duration;
+      background-color: #fff;
+      color: $color--button;
+      border: $border--thickness solid $color--button;
+      border-radius: 5px;
+    }
```

Comment:
1. transition: background-color $transition-duration; to &__button not to hover
2. change border-color instead of full border
3. border-radius: 5px; to  &__button not to hover